source/elements/oneMKL/source/domains/dft/config_params/workspace_placement.rst

```diff
@@ -0,0 +1,88 @@
+.. SPDX-FileCopyrightText: Codeplay Software
+..
+.. SPDX-License-Identifier: CC-BY-4.0
+
+.. _onemkl_dft_config_workspace_placement:
+
+Workspace placement
+--------------------------------------
+
+DFT implementations often require temporary storage for intermediate data whilst computing DFTs.
+This temporary storage is referred to as a *workspace*.
+Whilst this is managed automatically by default (``config_param::WORKSPACE_AUTOMATIC``), 
+it may be preferable to provide an external workspace (``config_param::WORKSPACE_EXTERNAL``) for the following reasons:
+
+* To reduce the number of GPU mallocs / frees
+* To reduce memory consumption
+
+For some backends and configurations, ``config_param::WORKSPACE_EXTERNAL`` may reduce performance.
+
+A typical workflow for using ``config_param::WORKSPACE_EXTERNAL`` is given in the section :ref:`onemkl_dft_typical_usage_of_workspace_external`.
+
+WORKSPACE_PLACEMENT
++++++++++++++++++++
+
+For ``config_param::WORKSPACE_PLACEMENT``, valid configuration values are ``config_value::WORKSPACE_AUTOMATIC`` and ``config_value::WORKSPACE_EXTERNAL``.
+
+.. container:: section
+
+  .. _onemkl_dft_config_value_workspace_automatic:
+
+  .. rubric:: WORKSPACE_AUTOMATIC
```

Comment:
Yes for `_onemkl_dft_config_value_workspace_automatic` - otherwise I get *WARNING: Content block expected for the "container" directive; none found.* The indent of `..rubric:` doesn't give any difference in the page that I can see, nor does it affect warnings. 

I'm new to writing in restructured text (thank you for your comments!), so do you think I should be formatting this differently. Perhaps I should get rid of the `.. container:: section`s, in which case no indents are needed?

Reply:
I do see a slight difference in the sub-title indentation when removing the indent before `..rubric::` on my end, but it might be browser-dependent: I honestly don't know.
I would not claim any expertise in restructured text myself, my own efforts were heavily guided by trial-and-error attempts and very specific web searches. I must say that the purpose of the `.. container:: section` directives remains very unclear to me. I opted for removing them in my own changes when that did not affect the end result, personally.

Reply:
I've removed the `..container` in [e139d35](https://github.com/oneapi-src/oneAPI-spec/pull/509/commits/e139d3564eb03f99e7d7d3eb5755e959b2a4414c), to follow the style of your previous PR. Hopefully this also removes the indentation.